adapters/smaato/smaatotest/params/race/video.json

```diff
@@ -0,0 +1,4 @@
+{
+  "publisherId": "1100042525",
+  "adspaceId": "130563103"
```

Comment:
Should this be `adbreakId` for video requests?

Reply:
no, a video imp via auction endpoint is fine and doesn't use adbreakId